docs/pages/web/dropdown.js

```diff
@@ -498,6 +501,16 @@ function CustomContentDropdownExample() {
     * \`Tab\` or \` Shift + Tab\` close the menu and move focus accordingly
           `}
         />
+        <MainSection.Subsection
+          title="Custom item content limitations"
+          description={`
+          If using custom content, do not include interactive elements, like a TextArea or Button. Because the Dropdown.Item and Dropdown.Link already act as buttons and links, respectively, they cannot include focusable elements as children. [Learn more about nested interactive controls](https://dequeuniversity.com/rules/axe/4.2/nested-interactive).
```

Comment:
grammar nits:
- no article before "Because _the_ Dropdown.Item and Dropdown.Link…"
- I'm like 95% sure there shouldn't be a comma before "respectively" (but the one after is fine)
- Unclear to me if the final "Learn more" sentence should end with a period, but I feel like it should at least match the "Learn more" in the `Custom item content` section below (I lean towards no period, but don't feel too strongly about it)